scripts/webhook.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+# Copyright (c) 2019-2020 Red Hat, Inc.
+# This program and the accompanying materials are made
+# available under the terms of the Eclipse Public License 2.0
+# which is available at https://www.eclipse.org/legal/epl-2.0/
+#
+# SPDX-License-Identifier: EPL-2.0
+#
+# Contributors:
+#   Red Hat, Inc. - initial API and implementation
+#
+
+NAMESPACE=""
+printHelp="false"
+while [[ "$#" -gt 0 ]]; do
+  case $1 in
+    '-n'|'--namespace') NAMESPACE="$2"; shift 1;;
+    '-h'|'--help') printHelp="true"; shift 0;;
+  esac
+  shift 1
+done
+
+if [[ "$printHelp" == "true" ]]; then
+  echo "Usage:"
```

Comment:
I think it makes sense to make it clearer that this script is for k8s-webhooks-certs via cert-manager.